cypress/integration/examples/network_requests.spec.js

```diff
@@ -0,0 +1,201 @@
+/// <reference types="cypress" />
+
+context('Network Requests', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/network-requests')
```

Comment:
The URL should be the Book Store app URL, not 'https://example.cypress.io/commands/network-requests'. Update the URL to visit the correct site.